apps/web/components/booking/BookingListItem.tsx

```diff
@@ -112,13 +128,46 @@ function BookingListItem(booking: BookingItem) {
 
   const startTime = dayjs(booking.startTime).format(isUpcoming ? "ddd, D MMM" : "D MMMM YYYY");
   const [isOpenRescheduleDialog, setIsOpenRescheduleDialog] = useState(false);
+  const [isOpenSetLocationDialog, setIsOpenLocationDialog] = useState(false);
+
+  const newMutation = useMutation(async (newLocation: string) => {
```

Comment:
NIT: a better description name for `newMutation` -> `setLocationMutation` | `postNewLocationMutation`